Description: use rimraf instead of fs.rm
 This permits compat with nodejs 12
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-02-24

--- a/tests/_test.js
+++ b/tests/_test.js
@@ -1,20 +1,21 @@
 const fs = require('fs')
 const os = require('os')
 const path = require('path')
+const rimraf = require('rimraf')
 const tape = require('tape')
 
 // for all practical purposes, this is a beforeEach and afterEach
 function test (desc, testFn) {
   tape(desc, function (t) {
     const testDir = path.join(os.tmpdir(), 'klaw-tests')
-    fs.rm(testDir, { recursive: true, force: true }, function (err) {
+    rimraf(testDir, function (err) {
       if (err) return t.end(err)
       fs.mkdir(testDir, function (err) {
         if (err) return t.end(err)
 
         const oldEnd = t.end
         t.end = function () {
-          fs.rm(testDir, { recursive: true, force: true }, function (err) {
+          rimraf(testDir, function (err) {
             err ? oldEnd.apply(t, [err]) : oldEnd.apply(t, arguments)
           })
         }
